src/components/App.jsx

```diff
@@ -61,20 +67,23 @@ function App() {
       while (history.includes(randomPrompt) || currentPrompt === randomPrompt) {
         randomPrompt = prompts[getRandomNumber()]
       }
-      if (currentPrompt === INITIALPROMPT) {
+      if (currentPrompt === INITIAL_PROMPT) {
         setCurrentPrompt(randomPrompt)
         saveToLocal('currentPrompt', randomPrompt)
+        setMustSpin(true)
       } else {
         setCurrentPrompt(randomPrompt)
         setHistory([...history, currentPrompt])
         saveToLocal('currentPrompt', randomPrompt)
         saveToLocal('promptHistory', [...history, currentPrompt])
+        setMustSpin(true)
       }
     } else {
-      setCurrentPrompt(LASTPROMPT)
-      saveToLocal('currentPrompt', LASTPROMPT)
+      setCurrentPrompt(LAST_PROMPT)
+      saveToLocal('currentPrompt', LAST_PROMPT)
       setHistory([...history, currentPrompt])
       saveToLocal('promptHistory', [...history, currentPrompt])
+      setMustSpin(false)
     }
   }
 }
```

Comment:
Within the code, it is easy to understand what happens under the respective conditions. You could consider outsourcing these functions for the sake of clarity inside the App component.

Reply:
Very good suggestions, thank you! I've put it on my to-do list 😊